src/three-components/ModelUtils.ts

```diff
@@ -87,6 +79,20 @@ export const cloneGltf = (gltf: Gltf): Gltf => {
     });
   }
 
+  const specularGlossiness =
+      gltf.parser.extensions['KHR_materials_pbrSpecularGlossiness'];
+  /**
+   * Creates a clone of the given material, and applies a patch to the
+   * shader program.
+   */
+  const cloneAndPatchMaterial = (material: Material): Material => {
+    const clone = (material as any).isGLTFSpecularGlossinessMaterial ?
+        specularGlossiness.cloneMaterial(material) :
+        material.clone();
+    clone.onBeforeCompile = updateShader;
+    return clone;
+  };
+
```

Comment:
Since we are moving this in here anyway, might as well move it inside the condition on line 99 so that we don't bother with it unless there is a scene in the glTF.

Reply:
SGTM